### DISCUSSION TITLE
How do I stop the "Would you like to update your Snapshots" dialog appearing?

### DISCUSSION THREAD

**Discussion by j-m** [orta/vscode-jest#639] [Q&A] [answered]
The appears every time VSCode saves regardless of whether the changes are related to the test.  
![image](https://user-images.githubusercontent.com/15386491/102251838-3ca49880-3efd-11eb-8f1a-5bc49390a638.png)
Is there a way to turn it off? I know this test is failing, and I don't want to replace the screenshots.

**Comment by cassus** [comment, marked as answer]
jest.enableSnapshotUpdateMessages setting controls this I believe

![Settings_—_org-mat-viz](https://user-images.githubusercontent.com/316826/115159762-f4f17180-a094-11eb-921d-796216371f81.jpg)

**Reply by j-m** [reply]
I no longer use this extension so I'll assume this is correct
